packages/@aws-cdk/cfnspec/lib/library-creation.ts

```diff
@@ -76,7 +76,7 @@ export async function createLibraryReadme(namespace: string, readmePath: string)
     'This module is part of the [AWS Cloud Development Kit](https://github.com/aws/aws-cdk) project.',
     '',
     '```ts',
-    `import ${module.moduleName.toLocaleLowerCase()} = require('${module.packageName}');`,
+    `import * as ${module.moduleName.toLocaleLowerCase().replace(/-/g, '_')} from '${module.packageName}';`,
```

Comment:
I think it might be overkill to add it here, since this is just a suggested name and can really be anything at this point. No need to add the `module` function from `generated-examples`, which is what I think you're referring to. Instead, I have added a more robust regex to ensure that the module name is always valid.